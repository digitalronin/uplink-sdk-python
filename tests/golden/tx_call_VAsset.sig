r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 82430133462376854190696667972169663448449762882327975154738791588541561947231
