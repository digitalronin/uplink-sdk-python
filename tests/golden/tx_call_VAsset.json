{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VAsset", 
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE04MjQzMDEzMzQ2MjM3Njg1NDE5MDY5NjY2Nzk3MjE2OTY2MzQ0ODQ0OTc2Mjg4MjMyNzk3NTE1NDczODc5MTU4ODU0MTU2MTk0NzIzMQ==", 
    "timestamp": 1231006505
}